.gitignore

```diff
@@ -56,5 +56,5 @@ target
 lib
 props-table
 **/__snapshots__/screen
-**/__snapshots__/diff
+**/__snapshots__/difflll
```

Comment:
Looks like maybe this got updated by accident?

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1214/commits/bf9a98f1133763653a236ea4ddfd48f5fe816267).